packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -140,17 +140,17 @@ declare module '@docusaurus/Head' {
   import type {HelmetProps} from 'react-helmet-async';
   import type {ReactNode} from 'react';
 
-  export type HeadProps = HelmetProps & {children: ReactNode};
+  export type Props = HelmetProps & {children: ReactNode};
 
-  const Head: (props: HeadProps) => JSX.Element;
+  const Head: (props: Props) => JSX.Element;
```

Comment:
Looking for another refactor?

I'd like to get rid of those extra useless consts: `export default function Head(props: Props)` should be enough, WDYT?

Reply:
Yeah, we have lots of these subtle inconsistencies. It's not a problem at all and I don't know any eslint rule to ensure that, so just letting it be

Reply:
no problem but still useful to simplify IMHO, let me know if you want to do it otherwise I'll do it later

Reply:
If you want to do that refactor later then sure. I _may_ be able to do it over the weekend, but overall I don't see much value in this.